crates/agglayer-tries/src/utils.rs

```diff
@@ -1,15 +1,1000 @@
-use agglayer_primitives::{keccak::keccak256_combine, Digest};
+use agglayer_primitives::Digest;
 
 /// Returns an array whose `i`th element is the root of an empty Merkle tree of
 /// depth `i`.
 #[inline]
-pub fn empty_hash_at_height<const DEPTH: usize>() -> [Digest; DEPTH] {
+pub const fn empty_hash_at_height<const DEPTH: usize>() -> [Digest; DEPTH] {
+    const {
+        assert!(DEPTH <= 192, "Only a depth of up to 192 is supported");
+    }
     let mut empty_hash_at_height = [Digest::ZERO; DEPTH];
-    for height in 1..DEPTH {
-        empty_hash_at_height[height] = keccak256_combine([
-            &empty_hash_at_height[height - 1],
-            &empty_hash_at_height[height - 1],
-        ]);
+    let mut i = 1; // 0 is already set to Digest::ZERO
+    while i < DEPTH {
+        empty_hash_at_height[i] = EMPTY_HASH_AT_192[i];
+        i += 1;
     }
     empty_hash_at_height
 }
+
```

Comment:
Would function that would return particular value of empty hash at height (not array) be useful somewhere in our code? As it is trivial to make it?